tests/.infra/crw-ci/pr-check/k8s/Jenkinsfile

```diff
@@ -207,28 +216,26 @@ pipeline {
         stage("Start Che") {
             steps {
                 script {
-                    echo "Patch Che custom-resource.yaml"
-                    sh """
-                        sed -i "s|cheImage: ''|cheImage: '${cheImageRepo}'|" $CUSTOM_RESOURCE_FILE_PATH
-                        sed -i "s|cheImageTag: 'nightly'|cheImageTag: '${mutableCheImageTag}'|" $CUSTOM_RESOURCE_FILE_PATH
-                        sed -i "s|cheLogLevel: INFO|cheLogLevel: DEBUG|" $CUSTOM_RESOURCE_FILE_PATH
-                        sed -i "s|ingressDomain: '192.168.99.101.nip.io'|ingressDomain: '\$(minikube ip).nip.io'|" $CUSTOM_RESOURCE_FILE_PATH
-
-                        # temporary workaround to https://github.com/eclipse/che/issues/16292
-                        sed -i "s/tlsSupport: true/tlsSupport: false/" $CUSTOM_RESOURCE_FILE_PATH  
-
-                        # TODO: "selfSignedCert: true" so as TLS support is enebled by default 
-                        # sed -i "s/selfSignedCert: false/selfSignedCert: true/" $CUSTOM_RESOURCE_FILE_PATH    
-                    """
+                    echo "Create custom-resource patch file"
+                    sh """cat > $CUSTOM_RESOURCE_PATCH_FILE_PATH <<EOL
```

Comment:
Typo:
```suggestion
                    sh """cat > $CUSTOM_RESOURCE_PATCH_FILE_PATCH <<EOL
```

Reply:
If you fix this, bear in mind it's used on multiple places.

Reply:
Good catch! Thx.

Reply:
It's actually PATH to custom resource patch file.

Reply:
I could probably simplify constant name to just `$CUSTOM_RESOURCE_PATCH_FILE`